test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -209,7 +195,7 @@ func schedulePods(config *testConfig) int32 {
 
 	// Bake in time for the first pod scheduling event.
 	for {
-		time.Sleep(50 * time.Millisecond)
+		//time.Sleep(50 * time.Millisecond)
```

Comment:
So, I tried multiple iterations and it worked even with this. So, commented that.(instead of deleting for future reference).